apache/superset
# refactor: sqleditorleftbar to functional

### SUMMARY
Converted the SqlEditorLeftBar class component to a functional component. Also removed unused functions from the component, specifically dbMutator and changeTable.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS

Go to the SQL Lab, and click on SQL Editor, the editor should open, and give you the option to select a database, schema, and other table schema. Choose the "examples" database and the "main" schema, both should be selected and the words "examples" and "main" appear appropriately in their selectors. From there you can write a query in the text editor and run it just to be sure. Run "SELECT * FROM birth_names" and you should see the query results render.

### ADDITIONAL INFORMATION
